cfme/tests/containers/test_smart_management.py

```diff
@@ -74,29 +82,39 @@ def wait_for_tag(obj_inst):
                                forced_streams=['5.7'],
                                unblock=lambda test_item: test_item.obj != Container)])
 def test_smart_management_add_tag(provider, test_item):
-
+    logger.info("Setting smart mgmt tag to {obj_type}".format(obj_type=test_item.obj.__name__))
     # validate no tag set to project
     if test_item.obj is ContainersProvider:
         obj_inst = provider
     else:
         obj_inst = test_item.obj.get_random_instances(provider, count=1).pop()
 
+    logger.info("Selected object is \"{obj_name}\"".format(obj_name=obj_inst.name))
```

Comment:
Maybe ``logger.debug``? Also use ``'..."{obj_name}"'`` so you don't need escape it.